Cargo.toml

```diff
@@ -12,13 +12,14 @@ path = "src/main.rs"
 
 [dependencies]
 anyhow = "1.0.75"
+bincode = "1.3.3"
 borsh = { version = "0.9.3" }
 clap = { version = "4.4.10", features = ["derive"] }
 dirs = "5.0.1"
 indexmap = { version = "2.1.0", features = ["serde"] }
 indicatif = "0.17.8"
-nifty-asset = { version = "0.1.0", features = ["serde"] }
-nifty-asset-types = { version = "0.1.0" }
+nifty-asset = { version = "0.4.0", features = ["serde"] }
+nifty-asset-types = { version = "0.4.0" }
```

Comment:
We don't need to import the `types` crates directly. It should be already reexported by the asset crate.